pmacApp/pmc/trajectory_scan_code.pmc

```diff
@@ -133,7 +133,7 @@ N102
         A_Vel = Next_A_Vel
     End If
     A_Adr = CalculatedBase + BufferLength
-    A_Vel_Adr = Z_Adr + BufferLength
+    A_Vel_Adr = CalculatedBase + 10*BufferLength
```

Comment:
Where does the 10 come from?

Reply:
Calculated is shifted 10 times from the `CalculatedBase`, being correspondent to the shift of Time/User buffer (both use the same address for the turbopmac) and the 9 axes position buffers.

I will change this part make it more readable, moving the addresses update to the bottom of this subprogram, as it is for the powerpmac.
That way, the  we can keep as
`A_Vel_Adr = Z_Adr + BufferLength`